cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -159,7 +159,7 @@ public KafkaBrokerConfigurationBuilder withStrimziMetricsReporter(StrimziReporte
             // Look at withUserConfiguration
             writer.println("metric.reporters=io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter");
             writer.println("prometheus.metrics.reporter.listener.enable=true");
-            writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:8080");
+            writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:9094");
```

Comment:
It's 9404, not 9094! Then, should we use `StrimziReporterMetricsModel.METRICS_PORT`?

Reply:
Yeah that makes sense. Like so?
`writer.println("prometheus.metrics.reporter.listener=http://0.0.0.0:" + StrimziReporterMetricsModel.METRICS_PORT);
`

Reply:
yes, that was my previous suggestion